test/spec/libraries/resource_runit_service_spec.rb

```diff
@@ -272,6 +272,15 @@
     resource.sv_verbose.should eq(true)
   end
 
+  it 'has a log_dir parameter that defaults to "/var/log/service_name"' do
+    resource.log_dir.should eq(::File.join('/var/log', resource.service_name))
```

Comment:
I think this should be:

```ruby
resource.log_dir.should eq('/var/log/getty.service')
```

I know we're using `resource.service_name` in other places in this spec, but since we set it, we know what it will be and should ensure that we're testing the right thing.